devices.js

```diff
@@ -760,6 +760,22 @@ const devices = [
         ],
         toZigbee: [],
     },
+    {
+        zigbeeModel: ['lumi.sensor_ht.agl02'],
+        model: 'WSDCGQ12LM',
+        vendor: 'Xiaomi',
+        description: 'Aqara temperature, humidity and pressure sensor',
+        supports: 'temperature, humidity and pressure',
+        fromZigbee: [fz.xiaomi_battery_3v, fz.xiaomi_temperature, fz.humidity, fz.WSDCGQ11LM_pressure,
```

Comment:
Can you remove/replace the following converters and check if pressure and temperature still work ok?
- `xiaomi_temperature` -> `temperature`
- `WSDCGQ11LM_pressure` -> `pressure`.
- Remove `WSDCGQ01LM_WSDCGQ11LM_interval` (not sure if this is used?)

Reply:
removed WSDCGQ01LM_WSDCGQ11LM_interval. replaced xiaomi_temperature -> temperature and WSDCGQ11LM_pressure -> pressure  can work properly